atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -471,6 +471,7 @@ def add_log_type(output):
         digests = self.get_digests()
         repositories = self.get_repositories(digests)
         arch = os.uname()[4]
+        tags = [image.tag for image in self.workflow.tag_conf.primary_images]
```

Comment:
It cannot, but you could create it as a set and then convert it to a list once modifications are done.